langs/jbang.yaml

```diff
@@ -0,0 +1,159 @@
+id: "jbang"
+aliases:
+  - "javac"
+name: "JBang"
+monacoLang: java
+
+install:
+  apt:
+    - default-jdk
+    - clang-format
+  manual: |
+    install -d "${pkg}/opt/jdt"
+
+    wget https://download.eclipse.org/jdtls/snapshots/jdt-language-server-latest.tar.gz
+    tar -xf jdt-language-server-latest.tar.gz -C "${pkg}/opt/jdt"
+
+
+
+
+main: "Main.java"
+template: |
+  public class Main {
+      public static void main(String[] args) {
+          System.out.println("Hello, world!");
+      }
+  }
+
+run: |
+    curl -Ls https://sh.jbang.dev | bash -s - Main.java
```

Comment:
We don't want to download something on every execution, as that will hurt performance. Can we download the language during installation, and run it from disk instead?

Reply:
Just temporary back.  That's just because didn't know how to get java available during install. Will fix it.